### ISSUE TITLE
Add OBS source management functions

### ISSUE THREAD

**Issue by YorVeX** [kostya9/NetObsBindings#6]
To really be able to do something meaningful with OBS I believe the two most important areas are source manipulation and the frontend API. This issue is regarding the first.

What we would need is the typical
- functions to retrieve data on sources, e.g.: obs_source_active(), obs_source_get_name(), obs_source_get_volume(), obs_source_get_width(), obs_source_get_height(), obs_source_get_settings()...
- functions to manipulate sources, e.g.: obs_source_set_name(), obs_source_set_volume(), obs_source_set_muted()...

So basically everything listed [here](https://obsproject.com/docs/reference-sources.html) for the generic overview or see [here](https://github.com/obsproject/obs-studio/blob/f6d0c63b1e33408319f29b829f0ec0af47f1fc6d/libobs/obs-source.c) for the actual implementation.

I would expect this should also be in ObsSource, which currently (in 0.0.1.17-alpha) provides only one function call:
![image](https://user-images.githubusercontent.com/528974/213035104-6d6f570c-0aa6-4a25-a643-8f404ad16712.png)

**Comment by kostya9** [comment]
> - functions to retrieve data on sources, e.g.: obs_source_active(), obs_source_get_name(), obs_source_get_volume(), obs_source_get_width(), obs_source_get_height(), obs_source_get_settings()
> - functions to manipulate sources, e.g.: obs_source_set_name(), obs_source_set_volume(), obs_source_set_muted().

You can find these functions in the Obs class.

I am basically following the convention of 1 class per header file. not sure that is the best way tbh...

**Comment by YorVeX** [comment]
OK makes sense, thanks!
